packages/monocdk-experiment/deps.js

```diff
@@ -0,0 +1,53 @@
+// +------------------------------------------------------------------------------------------------
```

Comment:
Wondering what the value here is of declaring `devDependencies` at all? Isn't that just another point of friction when a new lib is added and it's not registered here?

Reply:
To ensure this builds last (after all other modules)